src/WalmartScraper.py

```diff
@@ -24,7 +24,7 @@ def scrape_walmart():
     options.add_experimental_option("prefs", {"profile.default_content_setting_values.geolocation": 2})
 
     # Start Selenium webdriver
-    driver=webdriver.Chrome(service=Service(ChromeDriverManager().install()),options=options)
+    driver=webdriver.Chrome(service=Service(ChromeDriverManager(version='114.0.5735.90').install()),options=options)
```

Comment:
Same here, will this not work once chrome updates

Reply:
see previous